app/src/main/java/com/hedvig/app/feature/insurance/ui/detail/yourinfo/YourInfoAdapter.kt

```diff
@@ -44,13 +52,7 @@ class YourInfoAdapter :
                 if (data !is YourInfoModel.Header) {
                     return invalid(data)
                 }
-
-                root.setText(
-                    when (data) {
-                        YourInfoModel.Header.Details -> com.hedvig.app.R.string.CONTRACT_DETAIL_HOME_TITLE
-                        YourInfoModel.Header.Coinsured -> com.hedvig.app.R.string.CONTRACT_DETAIL_COINSURED_TITLE
-                    }
-                )
+                root.text = data.text
```

Comment:
Why did we move away from this structure? 😬

Reply:
I wanted to add another header, didnt think much about it. Your imperfection might be cleaner ill go back and use what was before